monitoring-bot/README.md

```diff
@@ -67,8 +67,8 @@ This is a bot to monitor some TFGrid functionalities here is a list:
     > Download the latest from the [releases page](https://github.com/threefoldtech/tfgrid-sdk-go/releases)
 
     ```bash
-    sudo cp monitoring-bot /usr/local/bin
-    monitoring-bot -e .env -w wallets.json
+    sudo cp tfgridmon /usr/local/bin
```

Comment:
this part of the documentation is using the release artifact which first need to use tar and the binary is called `monitoring-bot`